BazelExtensions/xcodeproject.bzl

```diff
@@ -180,7 +180,11 @@ def _install_xcode_project_impl(ctx):
         "sed -i '' \"s,\$SRCROOT,$SRCROOT,g\" "
         + output_proj
         + "/XCHammerAssets/bazel_build_settings.py",
-        "ln -sf $PWD/external $SRCROOT/external",
+        # If the external symlink exists already remove it and symlink to the
+        # bazel install root symlink. The execroot external directory might not
+        # have all material at one point
+        "test -h $SRCROOT/external && rm -rf $SRCROOT/external",
```

Comment:
Should the `rm` be recursive? It seems like that could be dangerous. `rm` or `rm -f` should be sufficient here, right?

Reply:
@jparise good suggestion - we don't expect to see a _directory_ at this path